Resources/Private/Templates/Standard/Accordion.html

```diff
@@ -1,23 +1,21 @@
-<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-bs-namespace-typo3-fluid="true">
 
 <f:layout name="Default"/>
 
 <f:section name="Main">
     <f:if condition="{children}">
-        <div id="accordion-{f:if(condition:data._LOCALIZED_UID, then:data._LOCALIZED_UID, else:data.uid)}">
+        <div id="accordion-{f:if(condition:data._LOCALIZED_UID, then:data._LOCALIZED_UID, else:data.uid)}" class="accordion">
```

Comment:
Kind of seems out-of-scope for me but i also don't see anything bad on this.

Reply:
@RozbehSharahi this was necessary because of a new bootstrap package and the changes in boostrap 5.x